#For attitude (roll, pitch, yaw) and controller outputs (3)
Header header
float64[3] rpy
float64[3] con_out
float64 target_thrust